textMining.py

```diff
@@ -0,0 +1,111 @@
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import requests
+
+def wikiInfo():
+    """
+        Originally worked with wikipedia to look at online text
+
+        returns: tuple of text strings from wikipedia articles
+    """
```

Comment:
Nice header comment. One improvement that I can think of is including what elements are inside the tuple that gets returned. It's good that you mentioned what data type the returned data are but I don't know what kind of information is included in the tuple (information that might be very helpful to understand how your code works as a whole).